From 542280651b453b04a49775bc619f6934281a495b Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <l.majewski@samsung.com>
Date: Tue, 8 Oct 2013 12:59:16 +0200
Subject: [PATCH 0850/1302] dts:exynos4412-redwood: Adjust REDWOOD dts to
 support changed cpufreq

Overriding the freq_table attribute inherited from SLP_PQ (Proxima).
Remove bindings for overlocking (now it is called boost).

Signed-off-by: Lukasz Majewski <l.majewski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4412-redwood.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/exynos4412-redwood.dts b/arch/arm/boot/dts/exynos4412-redwood.dts
index b24caa8..e03854e 100644
--- a/arch/arm/boot/dts/exynos4412-redwood.dts
+++ b/arch/arm/boot/dts/exynos4412-redwood.dts
@@ -139,9 +139,8 @@
 	};
 
 	cpufreq {
+		freq_table;
 		status = "okay";
-		overclocking = "okay";
-		max_overclocking_freq = <1500000>;
 	};
 
 	tmu@100C0000 {
-- 
1.8.3.2

